00-programming-fundamentals/06-submitting-projects-v1.md

```diff
@@ -3,7 +3,9 @@
 We utilize GitHub for all of our curriculum and project submissions. Before we learn how to use GitHub via the command line (terminal) we will need to be able to submit our projects using a process where we'll only use the web interface. Bookmark this page!
 
 ### Vocabulary
-**Fork**: A copy of the repository (right now, mostly just a `README` file) on your own personal GitHub account
+**Repository**: The term that GitHub uses for a collection of files that are all part of the same project. Every repository on GitHub is owned by an individual (e.g. [Hamled/dotnvim](https://github.com/Hamled/dotnvim)) or an organization (e.g. [github/hub](https://github.com/github/hub)).
+
+**Fork**: A copy of the repository (right now, mostly just a `README` file) on your own personal GitHub account. For example [AdaGold/Word-Guess](https://github.com/Ada-C7/Word-Guess) is the repository for the Word Guess project and [Hamled/Word-Guess](https://github.com/Hamled/Word-Guess) is Hamled's _fork_ of that repository.
```

Comment:
Actually, that's wrong. I originally was going to use the Ada-C7 one for the reasons stated above.

However I must have decided to use AdaGold anyways so that it would be more relevant to later cohorts? It seems the fork I created was directly from AdaGold, so I will change the link to reflect that.